lib/xcodeproj/constants.rb

```diff
@@ -4,27 +4,27 @@ module Xcodeproj
   module Constants
     # @return [String] The last known iOS SDK (stable).
     #
-    LAST_KNOWN_IOS_SDK = '14.0'
+    LAST_KNOWN_IOS_SDK = '15.4'
 
     # @return [String] The last known OS X SDK (stable).
     #
-    LAST_KNOWN_OSX_SDK = '10.15'
+    LAST_KNOWN_OSX_SDK = '12.3'
 
     # @return [String] The last known tvOS SDK (stable).
     #
-    LAST_KNOWN_TVOS_SDK = '14.0'
+    LAST_KNOWN_TVOS_SDK = '15.4'
 
     # @return [String] The last known watchOS SDK (stable).
     #
-    LAST_KNOWN_WATCHOS_SDK = '7.0'
+    LAST_KNOWN_WATCHOS_SDK = '8.5'
 
     # @return [String] The last known archive version to Xcodeproj.
     #
     LAST_KNOWN_ARCHIVE_VERSION = 1
 
     # @return [String] The last known Swift version (stable).
     #
-    LAST_KNOWN_SWIFT_VERSION = '5.0'
+    LAST_KNOWN_SWIFT_VERSION = '5.6'
```

Comment:
This might not actually be right? In Xcode, I see Swift 4, Swift 4.2, and Swift 5 as options, but nothing beyond Swift 5. I guess minor versions of Swift don't have their own compiler modes?

Reply:
I think this stays at 5 now, swift 4.2 was the last minor version to get its own compiler mode

Reply:
yeah this should stay at 5.0 for now